src/app/app.component.ts

```diff
@@ -265,8 +265,6 @@ export class AppComponent {
   }
 
   ngOnInit() {
-    // Ethnio survey 11-10-21
-    this._script.loadScript('ethnio-survey');
```

Comment:
Would you also delete `112:private _script`?

Reply:
Done